components/cli/pkg/artifacts/types.go

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2019 WSO2 Inc. (http:www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http:www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package artifacts
+
+type SystemComponent string
+
+const (
+	ApiManager       SystemComponent = "ApiManager"
+	IdentityProvider SystemComponent = "IdentityProvider"
+	Observability    SystemComponent = "Observability"
```

Comment:
Do we need to this? Can't we use `SystemComponent` in the runtime package?

Reply:
+1, will update